### DISCUSSION TITLE
Support for any arbitrary number of shades per color

### DISCUSSION THREAD

**Discussion by marnixhoh** [mantinedev/mantine#3464] [Feature requests]
Our design system has 11 shades per color - Mantine has a max of 10...

It would be great if there would no limit on the number of shades per color.

**Comment by rtivital** [comment]
You can add any number of shades. Mantine components will only use first ten of them.

**Reply by marnixhoh** [reply]
Awesome! Thank you for clarifying that and your super quick response :D

I absolutely love Mantine! Thank you for all that you're doing 👏
